### ISSUE TITLE
python finding the Icon for the app issue (and a fix)

### ISSUE THREAD

**Issue by Mervinpais** [AnonymousAmatuer/Memopad_Notepad#1]
basically you made mistake at line 9 where python tries to find an icon but it cant so it just crash's, using the "idle" version of python shows what error shows up:
`Traceback (most recent call last):
  File "C:\Users\mervi\Downloads\Memopad_Notepad-main\Memopad_Notepad-main\Memopad_version2.py", line 9, in <module>
    editor.iconbitmap(r'C:\Users\User\Downloads\notepad__1__M63_icon.ico') # Where did you store the image icon? You have to write in here.
  File "C:\Users\mervi\AppData\Local\Programs\Python\Python39\lib\tkinter\__init__.py", line 2080, in wm_iconbitmap
    return self.tk.call('wm', 'iconbitmap', self._w, bitmap)
_tkinter.TclError: bitmap "C:\Users\User\Downloads\notepad__1__M63_icon.ico" not defined`

Bascially Python says "uh oh! i cant find the icon file :("
the error on line 9 can be fixed by add a try catch statement
To understand in C# there is a try catch statement which is also in python (but its a try except statement)
the format 
C# version
`try 
{
}
catch 
{ 
}`
the try part means to make the code "try" to work but if it fails, the catch statement will start and you can add a message box saying there was an error and more

The python Version:
`
try:
 #whatever code
`
`
except:
 #if an error occured code here
`
TLDR
just replace line 9 with
`try:
`
`
    editor.iconbitmap(r'C:\Users\User\Downloads\notepad__1__M63_icon.ico')
`
`
except:
`
`
    print("Coudnt find icon file")
`

**Comment by AnonymousAmatuer** [comment]
@Mervinpais , I reuploaded the icon because I saw that the icon file is missing. I also simplified the name since it was to hard to comprehend. I also addressed your issue in my code in Line 9 
`editor.iconbitmap(r'C:\Users\User\Downloads\notepad icon.ico')`
` # Where did you store the image icon? You have to write in here.`
You can either omit the line in the code or you can paste the file location where you stored the icon. That way it will work. If that problem still persists, I will make a new file of it without the icon. Anyways, the default feather icon still looks attractive ! :D 
But as for the try and except, I know that, but it won't work properly.

Hope this helped your issue @Mervinpais

**Comment by AnonymousAmatuer** [comment]
@Mervinpais , I have uploaded a new file! It is called No Flaws Notepad. If you want to go without a icon go ahead! It has a default feather icon which is very attractive and still suits the name of the project!

**Comment by AnonymousAmatuer** [comment]
@Mervinpais , Thank You so much for addressing your issue. It is because of that I could see what is happening with the file. Now everyone can enjoy!

**Comment by AnonymousAmatuer** [comment]
Hey, if you want to rate my memopad look here @ http://bitly.ws/hkXG
But remember to log out from your email if your privacy concerned!

**Comment by SudoniousMacNCheese** [comment]
ok :D
